src/app/features/config/form-cfgs/sync-form.const.ts

```diff
@@ -47,11 +48,36 @@ export const SYNC_FORM: ConfigFormSection<SyncConfig> = {
             ? []
             : [{ label: SyncProvider.GoogleDrive, value: SyncProvider.GoogleDrive }]),
           { label: SyncProvider.WebDAV, value: SyncProvider.WebDAV },
-
-          ...(IS_ELECTRON
-            ? [{ label: SyncProvider.LocalFile, value: SyncProvider.LocalFile }]
-            : []),
+          ...((IS_ELECTRON || IS_ANDROID_WEB_VIEW) &&
+          androidInterface.grantFilePermissionWrapped &&
+          androidInterface.isGrantedFilePermission
+            ? []
+            : [{ label: SyncProvider.LocalFile, value: SyncProvider.LocalFile }]),
```

Comment:
If I am not mistaken this should be:
```suggestion
          ...((IS_ELECTRON || 
          (IS_ANDROID_WEB_VIEW &&
          androidInterface.grantFilePermissionWrapped &&
          androidInterface.isGrantedFilePermission))
            ? [{ label: SyncProvider.LocalFile, value: SyncProvider.LocalFile }]
            : []),
```

Reply:
Yes, I got a bit hasty. Fixed it.